components/hab/install.sh

```diff
@@ -285,13 +285,38 @@ do_install() {
 # Download location for the temporary files
 tmp_dir="${TMPDIR:-/tmp}/hab"
 
+# use stable channel by default
+channel="stable"
+
+# ## CLI Argument Parsing
+
+# Parse command line flags and options.
+while getopts "c:v:" opt; do
+  case $opt in
+    c)
+      channel=$OPTARG
+      ;;
+    v)
+      version=$OPTARG
+      ;;
+    \?)
+      print_help
```

Comment:
I don't think we have `print_help()` either, which we could replace with a really quick `echo "usage: $0 [-c CHANNEL] [-v HAB_VERSION]" line.